scripts/apps/authoring/authoring/directives/AuthoringDirective.js

```diff
@@ -754,6 +758,27 @@ export function AuthoringDirective(superdesk, superdeskFlags, authoringWorkspace
                 return lock.isLockedByMe($scope.item);
             };
 
+            /**
+            * On change content profile the default values from new content profile
+            * will overwrite the current values from item
+            *
+            * @function changeProfile
+            * @param {Object} item - ucurrent edited content items
+            */
+            $scope.changeProfile = function(item) {
```

Comment:
I agree with this, but this was the requirement to override anyway, even if the user already has set a different value.